src/StructuredLogViewer.Core/SearchResult.cs

```diff
@@ -8,6 +8,7 @@ public class SearchResult
     {
         public BaseNode Node { get; }
         public List<(string field, string match)> WordsInFields = new List<(string, string)>();
+        public Dictionary<string, string> SearchResultPaar = new Dictionary<string, string>();

```

Comment:
I'm a bit concerned with performance here. We can have a lot of results and allocating 96 bytes Dictionary for each seems a bit wasteful, since this is performance critical code. Maybe just keep two strings?